python/ray/serve/tests/unit/test_deployment_state.py

```diff
@@ -1308,29 +1311,121 @@ def test_deploy_new_config_new_version(mock_deployment_state_manager_full):
     )
 
 
-def test_stop_replicas_on_draining_nodes(mock_deployment_state_manager_full):
-    create_dsm, _, cluster_node_info_cache = mock_deployment_state_manager_full
+def test_stop_replicas_on_draining_nodes_1(mock_deployment_state_manager_full):
```

Comment:
instead of `_1`, `_2`, etc. please put these into a `class TestStopReplicasOnDrainingNodes` and give the methods more descriptive names :)